jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/NativeImagePathResource.java

```diff
@@ -0,0 +1,80 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.util.resource;
+
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * GraalVM Native-Image {@link Path} Resource.
+ */
+public class NativeImagePathResource extends PathResource
```

Comment:
Can this include the name `GraalVM` in the beginning?

aka `GraalVMNativeImagePathResource`

Reply:
I would prefer to avoid GraalVM specific classes.   

However, am I correct is thinking that this class is only needed to work around the Graal bug.  Once that bug is fixed, the factory can just create normal `PathResource` instances?

If so, then maybe even call this `GraalIssue5720PathResource` to make it very clear why it exists.